recipes/alm/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ALM" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "a3ee1b25a897ec8bd576dd00c948657c4faefbd42f2616bf5ad2ce79a3d8cda4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jochym/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - use_external_spglib.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - eigen
+  host:
+    - python
+    - blas 1.1 openblas  # [unix]
+    - numpy
+    - boost
+    - spglib >=1.11.1.2
+    - setuptools
+  run:
+    - python
+    - blas 1.1 openblas  # [unix]
+    - {{ pin_compatible('numpy') }}
+    - spglib >=1.11.1.2
+
+test:
+  source_files:
+    - test/*.dat
+    - test/*.py
+  requires:
+    - numpy
+  imports:
+    - alm
+  commands:
+    - cd test
+    - python Si_fitting.py
+    - python SiC_fitting.py
+
+about:
+  home: https://github.com/ttadano/ALM
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Software for calculating harmonic and anharmonic interatomic force constants in solids and molecules.'
+
+  description: |
+    ALM can be used for Extraction of harmonic and anharmonic force constants 
+    based on the supercell approach. It is applicable to any crystal structure
+    and low-dimensional systems. Features accurate treatment of translational 
+    and rotational invariance and Interface to VASP, Quantum-ESPRESSO, and xTAPP.
+  doc_url: http://alm.readthedocs.io/
+  dev_url: https://github.com/ttadano/ALM
+
+extra:
+  recipe-maintainers:
+    - jochym
```

Comment:
@ttadano, it looked like you were happy to have your package added to conda-forge. Would you like to be added as a co-maintainer?

Reply:
Yes. @jochym, could you add me to the list?